Nos/Views/Components/Author/AuthorListView.swift

```diff
@@ -39,7 +39,7 @@ struct AuthorListView: View {
         }
         .background(Color.appBg)
         .nosNavigationBar(title: .localizable.mention)
-        .onChange(of: searchTextObserver.debouncedText) { _, newValue in
+        .onChange(of: searchController.query) { _, newValue in
```

Comment:
The `SearchController` has its own internal logic to fire off searches whenever the `query` changes. It has some advanced logic like debouncing and de-duplicating requests to make sure we don't send too many. I think we should get rid of this `onChange` and the `search(for query:)` function in this file. 

Maybe eventually we will need an `onChange(of: searchController.authorResults)` to do some custom sorting of the results and add participants in the thread. But for this PR I think we could even bind the `ForEach` directly to `searchController.authorResults` and get rid of the `filteredAuthors` array.